hostinfo/packagetype_container.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//gobuild linux && ts_package_container
+
+package hostinfo
+
+func init() {
+	packageType = packageTypeContainer
+}
+
+func packageTypeContainer() string {
+	return "container"
```

Comment:
nit: maybe `ts_container` or `official_container`?

Reply:
None of the other values contain "ts_" or "official", though....

<img width="451" alt="Screenshot 2024-07-17 at 10 54 33 AM" src="https://github.com/user-attachments/assets/6b889ba6-a22e-4280-9fbe-9358fc046650">